pylhc_submitter/htc/utils.py

```diff
@@ -186,7 +187,16 @@ def write_bash(
                 f.write(f"{SHEBANG}\n") 
             if output_dir is not None:
                 f.write(f"mkdir {str(output_dir)}\n")
-            f.write(f"{exec_path}{str(job_dir / job[COLUMN_JOB_FILE])}{cmds}\n")
+            f.write(f"{exec_path}")
+
+            if is_mask_file(mask):
+                f.write(f"{str(job_dir / job[COLUMN_JOB_FILE])}")
+            elif is_mask_string(mask):

```

Comment:
Shouldn't this just be `else` ? And then you don't need the second function at all...